### ISSUE TITLE
Installation of 1.2.0 fails during symlinking; fix by symlinking /usr/local to boxen

### ISSUE THREAD

**Issue by jakubholynet** [boxen/puppet-atom#4]
Installation of Atom 1.2.0 failed with Permission denied during sumlinking:

```
Error: /Stage[main]/Atom/Package[atom]/ensure: change from absent to present failed: Execution of 'brew cask install atom --appdir=/Applications' returned 1: ==> Downloading https://atom.io/download/mac
Already downloaded: /Library/Caches/Homebrew/atom-latest
==> Symlinking App 'Atom.app' to '/Applications/Atom.app'
Error: Permission denied @ dir_s_mkdir - /usr/local
==> Symlinking Binary 'apm' to '/usr/local/bin/apm'

  Most likely, this means you have an outdated version of homebrew-cask. Please run:

      brew update && brew upgrade brew-cask && brew cleanup && brew cask cleanup

  If this doesn’t fix the problem, please report this bug:

      https://github.com/caskroom/homebrew-cask/issues

/opt/boxen/homebrew/Cellar/ruby/2.2.2/lib/ruby/2.2.0/fileutils.rb:252:in `mkdir'
/opt/boxen/homebrew/Cellar/ruby/2.2.2/lib/ruby/2.2.0/fileutils.rb:252:in `fu_mkdir'
/opt/boxen/homebrew/Cellar/ruby/2.2.2/lib/ruby/2.2.0/fileutils.rb:226:in `block (2 levels) in mkdir_p'
/opt/boxen/homebrew/Cellar/ruby/2.2.2/lib/ruby/2.2.0/fileutils.rb:224:in `reverse_each'
/opt/boxen/homebrew/Cellar/ruby/2.2.2/lib/ruby/2.2.0/fileutils.rb:224:in `block in mkdir_p'
/opt/boxen/homebrew/Cellar/ruby/2.2.2/lib/ruby/2.2.0/fileutils.rb:210:in `each'
/opt/boxen/homebrew/Cellar/ruby/2.2.2/lib/ruby/2.2.0/fileutils.rb:210:in `mkdir_p'
/opt/boxen/homebrew/Cellar/ruby/2.2.2/lib/ruby/2.2.0/pathname.rb:564:in `mkpath'
/opt/boxen/homebrew/Cellar/brew-cask/0.53.3/rubylib/hbc/artifact/symlinked.rb:11:in `create_filesystem_link'
```

A possible solution is to add this to e.g. `modules/people/manifests/githubUserId.pp`:
 
```ruby
  # Symlink to fix Atom install, see
  # https://github.com/caskroom/homebrew-cask/issues/4716
  file { '/usr/local': 
    ensure => link,
    target => '/opt/boxen',
  }
```

Related info:

* There was no `/user/local/` (because I removed it to get rid of the pre-existing Homebrew installation - perhaps that is the problem? [uninstalling H. via its uninstall script failed so I had to do this manually])

**Comment by kn0ll** [comment]
had same issue. file directive fixed it for me as well.

**Comment by timdorr** [comment]
I put a PR to fix #5
